domain_access/src/DomainAccessManager.php

```diff
@@ -41,6 +41,7 @@ public function __construct(DomainLoaderInterface $loader, DomainNegotiatorInter
    * @inheritdoc
    */
   public function getAccessValues(EntityInterface $entity, $field_name = DOMAIN_ACCESS_FIELD) {
+    /** @var FieldableEntityInterface $entity */
```

Comment:
This is a case where I don't know that commenting the variable adds any clarity, so I think we just live with the uncertainty that core hands us and don't bother annotating this variable (which is type-hinted in the line above).